docs/dep-baseline.md

```diff
@@ -23,14 +23,14 @@
 
 以下是 `ruyi` 依赖的 **架构无关** 软件包在一些发行版的提供情况：
 
-| 发行版版本          | arpy  | certifi    | fastjsonschema | jinja2 | packaging | requests | rich   | semver | tomlkit | typing\_extensions |
-|---------------------|-------|------------|----------------|--------|-----------|----------|--------|--------|---------|--------------------|
-| Debian 12           | 1.1.1 | 2020.6.20  | 2.16.3         | 3.0.3  | 21.3      | 2.25.1   | 11.2.0 | 2.10.2 | 0.9.2   | 3.10.0.2           |
-| Fedora 39           | 2.3.0 | 2023.05.07 | 2.16.3         | 3.1.4  | 23.1      | 2.28.2   | 13.5.2 | 3.0.2  | 0.11.4  | 4.12.2             |
-| Fedora 40           | 2.3.0 | 2023.05.07 | 2.18.0         | 3.1.4  | 23.2      | 2.31.0   | 13.7.0 | 3.0.2  | 0.12.3  | 4.12.2             |
-| Fedora 41           | 2.3.0 | 2023.05.07 | 2.19.1         | 3.1.4  | 24.1      | 2.32.3   | 13.7.1 | 3.0.2  | 0.12.4  | 4.12.2             |
-| Fedora 42           | 2.3.0 | 2024.08.30 | 2.21.1         | 3.1.6  | 24.2      | 2.32.3   | 13.9.4 | 3.0.2  | 0.13.2  | 4.12.2             |
-| openEuler 24.03 LTS | :x:   | 2024.2.2   | 2.19.1         | 3.1.3  | 23.2      | 2.31.0   | 13.7.1 | 3.0.2  | 0.12.3  | 4.10.0             |
-| openEuler 24.09     | :x:   | 2024.7.4   | 2.19.1         | 3.1.3  | 23.2      | 2.31.0   | 13.7.1 | 3.0.2  | 0.12.3  | 4.12.2             |
-| Ubuntu 22.04 LTS    | 1.1.1 | 2020.6.20  | 2.15.1         | 3.0.3  | 21.3      | 2.25.1   | 11.2.0 | 2.10.2 | 0.9.2   | 3.10.0.2           |
-| Ubuntu 24.04 LTS    | 1.1.1 | 2023.11.17 | 2.19.0         | 3.1.2  | 24.0      | 2.31.0   | 13.7.1 | 2.10.2 | 0.12.4  | 4.10.0             |
+| 发行版版本          | arpy  | certifi    | fastjsonschema | jinja2 | packaging | requests | rich   | semver | tomlkit | typing\_extensions | argcomplete |
+| ------------------- | ----- | ---------- | -------------- | ------ | --------- | -------- | ------ | ------ | ------- | ------------------ |
```

Comment:
The columns are apparently alphabetically sorted so you should move it before `arpy`...